src/index.html

```diff
@@ -10,13 +10,17 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Stop watch</title>
+    <title>Stop stopwatch</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The 'href' attribute should link to a .css file, not a .scss file. SCSS files are preprocessed into CSS, and only CSS files should be linked in HTML.